Cargo.toml

```diff
@@ -10,3 +10,6 @@ itertools = "*"
 regex = "*"
 lazy_static = "*"
 bstr = "*"
+
+[dev-dependencies]
+pretty_assertions = "*"
```

Comment:
unrelated to the PR, but will this eventually end up on crates.io? And if so do they allow wildcard dependency versions?

Reply:
Hmm. Good question. Initially we'll probably only be distributing a binary, which I don't have experience with on crates.io. Not sure if it gives allowances to binaries on the manifest if they also contain a lockfile.

Reply:
Assuming the library uses semantic versioning, I usually wildcard the patch-version, and lock the major and minor version. In this case, it's a dev-dependency for the unittests which decreases the risk of a complete wildcard.

Important in this context is whether you check in the `Cargo.toml` file (recommended for binaries) or not (recommended for libraries). If you don't check in the lock-file, you have a higher risk of build-failures due to changes in external dependencies.